frontend/src/index.tsx

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { ClassPrefix } from '@carbon/react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import { AuthProvider } from './contexts/AuthContext';
 import { ThemePreference } from './utils/ThemePreference';
+import { env } from './env';
+import makeServer from './mock-server/server';
+
+const isDevEnv = env.REACT_APP_NRSPARWEBAPP_VERSION === 'dev' || env.REACT_APP_NRSPARWEBAPP_VERSION === 'test';
+
+if (env.REACT_APP_NRSPARWEBAPP_VERSION === 'dev') {
```

Comment:
Shouldn't we check for `isDevEnv` in this `if`? Now that the mock-server can be used alongside with the real API, I think we can leave the mocked features on the test environment as well, so testing users can check how the feature will function once it is connected to the real API.

Reply:
good point, [changed](https://github.com/bcgov/nr-spar-webapp/pull/85/commits/84e0f330899b1ef7ab20fe19c579684e0f21f8f6)